### DISCUSSION TITLE
CheckboxCard addon does not change bgColor to match parent when is selected

### DISCUSSION THREAD

**Discussion by BBezzola** [chakra-ui/chakra-ui#9277] [Q&A] [answered]
I'm creating a `CheckboxCard` with `variant="subtle"` but it seems that the addon background does not change like the `CheckboxCard` parent when it is selected

<img width="501" alt="image" src="https://github.com/user-attachments/assets/36c08c0d-b7b6-41fe-90e4-b52b6750f859">

See [playground](https://codesandbox.io/p/sandbox/lucid-river-6v2r9s?file=%2FApp.tsx)

Is this the expected behaviour? 
I'd expect the addon to match the parent bg when selected

**Comment by segunadebayo** [comment, marked as answer]
That's expected. The subtle variant only colors `control` when checked. If you want the addon to match, add `_checked` styles for `addon` in the recipe.
